modules/distribution/src/repository/conf/security/authenticators.xml

```diff
@@ -22,39 +24,43 @@
-		<Priority>10</Priority>
-		<Config>
-			<Parameter name="isRemote">false</Parameter>
-			<Parameter name="hostURL">https://localhost:9443</Parameter>
-		</Config>
-	</Authenticator>
-
-	<!-- Authenticator Configurations for TokenUIAuthenticator -->
-	<Authenticator name="TokenUIAuthenticator" disabled="false">
-		<Priority>5</Priority>
-	</Authenticator>
-
-	<!-- Authenticator Configurations for SAML2SSOAuthenticator -->
-	<Authenticator name="SAML2SSOAuthenticator" disabled="true">
-		<Priority>10</Priority>
-		<Config>
-			<Parameter name="LoginPage">/carbon/admin/login.jsp</Parameter>
-			<Parameter name="ServiceProviderID">carbonServer</Parameter>
-			<Parameter name="IdentityProviderSSOServiceURL">https://localhost:9443/samlsso</Parameter>
-			<Parameter name="NameIDPolicyFormat">urn:oasis:names:tc:SAML:1.1:nameid-format:unspecified</Parameter>
-		</Config>
-
-		<!-- If this authenticator should skip any URI from authentication, specify it under "SkipAuthentication"
-		<SkipAuthentication>
-				<UrlContains></UrlContains>
-			</SkipAuthentication> -->
-
-		<!-- If this authenticator should skip any URI from session validation, specify it under "SkipAuthentication
-			<SkipSessionValidation>
-				<UrlContains></UrlContains>
-			</SkipSessionValidation> -->
-	</Authenticator>
+    <!-- Authenticator Configurations for OAuthAuthenticator -->
+    <Authenticator name="BackEndOAuthAuthenticator" disabled="false">
+        <Priority>10</Priority>
+        <Config>
+            <Parameter name="isRemote">false</Parameter>
+            <Parameter name="hostURL">https://localhost:9443</Parameter>
+        </Config>
+    </Authenticator>
+
+    <!-- Authenticator Configurations for TokenUIAuthenticator -->
+    <Authenticator name="TokenUIAuthenticator" disabled="false">
+        <Priority>5</Priority>
+    </Authenticator>
+
+    <!-- Authenticator Configurations for SAML2SSOAuthenticator -->
+    <Authenticator name="SAML2SSOAuthenticator" disabled="true">
+        <Priority>10</Priority>
+        <Config>
+            <Parameter name="LoginPage">/carbon/admin/login.jsp</Parameter>
+            <Parameter name="ServiceProviderID">carbonServer</Parameter>
+            <Parameter name="IdentityProviderSSOServiceURL">https://localhost:9443/samlsso
+            </Parameter>
+            <Parameter name="NameIDPolicyFormat">
+                urn:oasis:names:tc:SAML:1.1:nameid-format:unspecified
+            </Parameter>
+        </Config>
+
+        <!-- If this authenticator should skip any URI from authentication, specify it under "SkipAuthentication"
+        <SkipAuthentication>
+                <UrlContains></UrlContains>
+            </SkipAuthentication> -->
+
+        <!-- If this authenticator should skip any URI from session validation, specify it under "SkipAuthentication
+            <SkipSessionValidation>
+                <UrlContains></UrlContains>
+            </SkipSessionValidation> -->
```

Comment:
Already fixed in commit https://github.com/wso2-incubator/product-iot-server/commit/febdc36cba8b94c3ebcaba27947893ebb2865513 as per the PrabathA's comment.